src/elli_http.erl

```diff
@@ -692,6 +692,20 @@ content_length(Headers, Body)->
 %% PATH HELPERS
 %%
 
+-ifdef(OTP_RELEASE).
```

Comment:
`?OTP_RELEASE` was added in OTP 21, but we'll want to use `uri_string:parse/1` only for OTP >= 22.

Reply:
Ah, damn, ok.